common/src/runMain.ts

```diff
@@ -34,8 +32,7 @@ export const runMain = async (output: Output, logger: Logger, config: Config, st
       logAssertionResults(logger, finishedRun.assertions);
       logResult(logger, config, startedRun, finishedRun);
 
-      await output.set("run_status_code", finishedRun.statusCode);
-      await output.set("run_status_name", finishedRun.statusName);
+      await output.set("run_status_name", RunStatusDisplayNames[finishedRun.status]);
```

Comment:
Breaking change, may be discussed.

My opinion is that we may want to upgrade the version major:
- No longer compatible with Self-Hosted
- Slight rework of this output, but it should not be useful as it was.

We chose to fork and make another repository for teamcity/jenkins/etc. but in this repo we do not have mentions of frontline, even if its currently compatible.

WDYT @Isammoc ?

Reply:
New major version incoming with this PR, breaking change ok.